docs/docs/cli.md

```diff
@@ -187,4 +228,13 @@ See the documentation on [phrase-specific features] for more information.
 sku translations pull
 ```
 
+### `translations validate`
+
+Validate translations defined in `.vocab` directories.
```

Comment:
We should probably define what this actually means. Probably worth documenting this more thoroughly in `vocab` and then just linking to it. Happy to add this as-is for now though.